config.template.py

```diff
@@ -45,5 +45,3 @@
 
 PORT = 5666  # The application port nginx proxy is passing to
 APP_BASE_URL = "http://{}:{}".format(HOST, PORT)
-
-PAGE_SIZE = 12
```

Comment:
If there are no `args` in the URL does sqlalchemy also provide default value to `size`? Depending on that we don't need this as a fallback and we can remove it

Reply:
Yes, exactly!